utils/dvtm/Makefile

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright (C) 2009-2016 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=dvtm
+PKG_VERSION:=0.15
+PKG_RELEASE:=1
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/dvtm-$(PKG_VERSION)
+PKG_SOURCE:=dvtm-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=http://www.brain-dump.org/projects/dvtm/
+PKG_MD5SUM:=887e162a3abe2ad8e86caefab20cdd63
+PKG_MAINTAINER:=Yunfan Jiang<jyf1987@gmail.com>
+
+PKG_LICENSE:=MIT/X
+PKG_LICENSE_FILES:=COPYING
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/dvtm
+	SECTION:=utils
+	CATEGORY:=Utilities
+	DEPENDS:=+libncursesw
+	TITLE:=dynamic virtual terminal manager
+	URL:=http://www.brain-dump.org/projects/dvtm/
+endef
```

Comment:
i have build from the Makefile and tested on my mt7620a based device.
i've no idea if it will cause problem on other devices
sorry for the non-formal